### PR TITLE
Add option to sort via semver before determin latest tag

### LINKED ISSUE #145: Misspredict stop of last release
https://github.com/woodpecker-ci/woodpecker/pull/3938#issuecomment-2238553397

we had an v2.7.0 release (next was a v2.6.1 but not marked as latest) and now a 3.0.0 is (currently if breaking changes are not reverted) expected.

anyway the new 3.0.0 has all the pulls listed that are already part in v2.7.0

### PR DESCRIPTION
**Pull request by 6543**
close  #145

image

### CODE REVIEW

File: src/utils/config.ts [hunk 1]
```diff
@@ -65,6 +65,7 @@ export const defaultUserConfig: UserConfig = {
   skipLabels: ['skip-release', 'skip-changelog', 'regression'],
   skipCommitsWithoutPullRequest: true,
   commentOnReleasedPullRequests: true,
+  sortTags: true,
```
Comment (anbraten) [inline src/utils/config.ts:68]: @6543 Why is there an option to disable this?

Comment (6543) [reply src/utils/config.ts:68]: if you want the old behavior instead ... you could do so

Comment (6543) [reply src/utils/config.ts:68]: @anbraten should I drop the config option and make it default?

Comment (anbraten) [reply src/utils/config.ts:68]: I think its kind of a bugfix and I can't think why someone would want the other behavior, so I would just remove it.

**Pull request merged**